api/pb/android/threads/src/test/java/io/textile/threads/ClientUnitTest.java

```diff
@@ -24,7 +26,7 @@
 public class ClientUnitTest {
 
     static Client client;
-    static String dbId = "bafk7ayo2xuuafgx6ubbcn2lro3s7oixgujdda6shv4";
+    static String dbId = "AVXwYdq9KAKa/qBCJulxduX3IuaiRjB6R68=";
```

Comment:
Since we don't have a Java Thread ID lib, I'm just hard-hardcoding a base64 representation of a raw thread ID bytes.